doc/api/cli.md

```diff
@@ -178,6 +178,14 @@ added: v8.0.0
 Enable loading native modules compiled with the ABI-stable Node.js API (N-API)
 (experimental).
 
+## `--abort-on-uncaught-exception`
+<!-- YAML
+added: v0.10
+-->
+
+Aborting instead of exiting causes a core file to be generated for post-mortem
```

Comment:
Suggestion (feel free to take it or ignore it):

Abort the process and generate a core file if an uncaught exception occurs. The core file can be used for post-mortem analysis using a debugger such as `lldb`, `gdb`, and `mdb`.